[2025-09-07T00:07:27.825+0000] {processor.py:186} INFO - Started process (PID=25557) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T00:07:27.826+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T00:07:27.827+0000] {logging_mixin.py:190} INFO - [2025-09-07T00:07:27.827+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T00:07:28.164+0000] {logging_mixin.py:190} INFO - [2025-09-07T00:07:28.163+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T00:07:28.165+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T00:07:28.173+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.351 seconds
[2025-09-07T00:22:36.680+0000] {processor.py:186} INFO - Started process (PID=25600) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T00:22:36.681+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T00:22:36.683+0000] {logging_mixin.py:190} INFO - [2025-09-07T00:22:36.683+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T00:22:37.002+0000] {logging_mixin.py:190} INFO - [2025-09-07T00:22:37.001+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T00:22:37.002+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T00:22:37.010+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.334 seconds
[2025-09-07T00:55:22.504+0000] {processor.py:186} INFO - Started process (PID=25645) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T00:55:22.504+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T00:55:22.505+0000] {logging_mixin.py:190} INFO - [2025-09-07T00:55:22.505+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T00:55:22.825+0000] {logging_mixin.py:190} INFO - [2025-09-07T00:55:22.823+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T00:55:22.825+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T00:55:22.834+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.333 seconds
[2025-09-07T02:23:13.705+0000] {processor.py:186} INFO - Started process (PID=25686) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T02:23:13.706+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T02:23:13.708+0000] {logging_mixin.py:190} INFO - [2025-09-07T02:23:13.708+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T02:23:14.025+0000] {logging_mixin.py:190} INFO - [2025-09-07T02:23:14.024+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T02:23:14.026+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T02:23:14.034+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.333 seconds
[2025-09-07T02:23:44.727+0000] {processor.py:186} INFO - Started process (PID=25729) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T02:23:44.727+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T02:23:44.729+0000] {logging_mixin.py:190} INFO - [2025-09-07T02:23:44.729+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T02:23:45.056+0000] {logging_mixin.py:190} INFO - [2025-09-07T02:23:45.055+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T02:23:45.057+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T02:23:45.065+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.342 seconds
[2025-09-07T03:12:35.545+0000] {processor.py:186} INFO - Started process (PID=25774) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T03:12:35.546+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T03:12:35.548+0000] {logging_mixin.py:190} INFO - [2025-09-07T03:12:35.547+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T03:12:35.927+0000] {logging_mixin.py:190} INFO - [2025-09-07T03:12:35.925+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T03:12:35.927+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T03:12:35.936+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.396 seconds
[2025-09-07T04:54:40.944+0000] {processor.py:186} INFO - Started process (PID=25815) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T04:54:40.945+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T04:54:40.946+0000] {logging_mixin.py:190} INFO - [2025-09-07T04:54:40.946+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T04:54:41.341+0000] {logging_mixin.py:190} INFO - [2025-09-07T04:54:41.340+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T04:54:41.342+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T04:54:41.348+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.407 seconds
[2025-09-07T06:14:16.953+0000] {processor.py:186} INFO - Started process (PID=25858) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T06:14:16.953+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T06:14:16.955+0000] {logging_mixin.py:190} INFO - [2025-09-07T06:14:16.955+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T06:14:17.353+0000] {logging_mixin.py:190} INFO - [2025-09-07T06:14:17.352+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T06:14:17.354+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T06:14:17.361+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.412 seconds
[2025-09-07T06:14:47.572+0000] {processor.py:186} INFO - Started process (PID=25901) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T06:14:47.572+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T06:14:47.574+0000] {logging_mixin.py:190} INFO - [2025-09-07T06:14:47.574+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T06:14:47.971+0000] {logging_mixin.py:190} INFO - [2025-09-07T06:14:47.970+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T06:14:47.971+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T06:14:47.981+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.413 seconds
[2025-09-07T07:16:49.682+0000] {processor.py:186} INFO - Started process (PID=25944) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T07:16:49.684+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T07:16:49.686+0000] {logging_mixin.py:190} INFO - [2025-09-07T07:16:49.686+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T07:16:50.087+0000] {logging_mixin.py:190} INFO - [2025-09-07T07:16:50.086+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T07:16:50.088+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T07:16:50.095+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.416 seconds
[2025-09-07T07:17:20.890+0000] {processor.py:186} INFO - Started process (PID=25987) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T07:17:20.891+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T07:17:20.892+0000] {logging_mixin.py:190} INFO - [2025-09-07T07:17:20.892+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T07:17:21.291+0000] {logging_mixin.py:190} INFO - [2025-09-07T07:17:21.290+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T07:17:21.292+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T07:17:21.298+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.413 seconds
[2025-09-07T08:15:16.130+0000] {processor.py:186} INFO - Started process (PID=26030) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T08:15:16.131+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T08:15:16.133+0000] {logging_mixin.py:190} INFO - [2025-09-07T08:15:16.133+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T08:15:16.538+0000] {logging_mixin.py:190} INFO - [2025-09-07T08:15:16.536+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T08:15:16.538+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T08:15:16.545+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.420 seconds
[2025-09-07T08:15:46.704+0000] {processor.py:186} INFO - Started process (PID=26075) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T08:15:46.705+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T08:15:46.706+0000] {logging_mixin.py:190} INFO - [2025-09-07T08:15:46.706+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T08:15:47.094+0000] {logging_mixin.py:190} INFO - [2025-09-07T08:15:47.093+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T08:15:47.095+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T08:15:47.102+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.401 seconds
[2025-09-07T09:07:42.772+0000] {processor.py:186} INFO - Started process (PID=26116) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T09:07:42.773+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T09:07:42.776+0000] {logging_mixin.py:190} INFO - [2025-09-07T09:07:42.775+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T09:07:43.205+0000] {logging_mixin.py:190} INFO - [2025-09-07T09:07:43.203+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T09:07:43.205+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T09:07:43.212+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.443 seconds
[2025-09-07T09:16:43.985+0000] {processor.py:186} INFO - Started process (PID=26159) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T09:16:43.985+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T09:16:43.987+0000] {logging_mixin.py:190} INFO - [2025-09-07T09:16:43.987+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T09:16:44.439+0000] {logging_mixin.py:190} INFO - [2025-09-07T09:16:44.438+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T09:16:44.439+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T09:16:44.446+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.465 seconds
[2025-09-07T09:51:32.769+0000] {processor.py:186} INFO - Started process (PID=26202) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T09:51:32.771+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T09:51:32.773+0000] {logging_mixin.py:190} INFO - [2025-09-07T09:51:32.773+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T09:51:33.163+0000] {logging_mixin.py:190} INFO - [2025-09-07T09:51:33.162+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T09:51:33.164+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T09:51:33.171+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.405 seconds
[2025-09-07T09:52:04.060+0000] {processor.py:186} INFO - Started process (PID=26247) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T09:52:04.061+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T09:52:04.062+0000] {logging_mixin.py:190} INFO - [2025-09-07T09:52:04.062+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T09:52:04.379+0000] {logging_mixin.py:190} INFO - [2025-09-07T09:52:04.377+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T09:52:04.379+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T09:52:04.388+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.331 seconds
[2025-09-07T09:52:35.160+0000] {processor.py:186} INFO - Started process (PID=26288) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T09:52:35.160+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T09:52:35.162+0000] {logging_mixin.py:190} INFO - [2025-09-07T09:52:35.162+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T09:52:35.502+0000] {logging_mixin.py:190} INFO - [2025-09-07T09:52:35.501+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T09:52:35.503+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T09:52:35.511+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.355 seconds
[2025-09-07T10:08:00.064+0000] {processor.py:186} INFO - Started process (PID=26331) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T10:08:00.065+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T10:08:00.066+0000] {logging_mixin.py:190} INFO - [2025-09-07T10:08:00.066+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T10:08:00.386+0000] {logging_mixin.py:190} INFO - [2025-09-07T10:08:00.385+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T10:08:00.387+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T10:08:00.395+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.334 seconds
[2025-09-07T10:08:31.286+0000] {processor.py:186} INFO - Started process (PID=26376) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T10:08:31.287+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T10:08:31.288+0000] {logging_mixin.py:190} INFO - [2025-09-07T10:08:31.288+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T10:08:31.632+0000] {logging_mixin.py:190} INFO - [2025-09-07T10:08:31.631+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T10:08:31.632+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T10:08:31.643+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.361 seconds
[2025-09-07T10:16:02.038+0000] {processor.py:186} INFO - Started process (PID=26417) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T10:16:02.039+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T10:16:02.041+0000] {logging_mixin.py:190} INFO - [2025-09-07T10:16:02.040+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T10:16:02.360+0000] {logging_mixin.py:190} INFO - [2025-09-07T10:16:02.359+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T10:16:02.361+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T10:16:02.369+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.334 seconds
[2025-09-07T10:31:51.895+0000] {processor.py:186} INFO - Started process (PID=26460) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T10:31:51.896+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T10:31:51.897+0000] {logging_mixin.py:190} INFO - [2025-09-07T10:31:51.897+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T10:31:52.217+0000] {logging_mixin.py:190} INFO - [2025-09-07T10:31:52.215+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T10:31:52.218+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T10:31:52.226+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.335 seconds
[2025-09-07T11:39:04.605+0000] {processor.py:186} INFO - Started process (PID=26503) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T11:39:04.606+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T11:39:04.607+0000] {logging_mixin.py:190} INFO - [2025-09-07T11:39:04.607+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T11:39:04.922+0000] {logging_mixin.py:190} INFO - [2025-09-07T11:39:04.920+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T11:39:04.922+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T11:39:04.930+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.328 seconds
[2025-09-07T13:02:08.962+0000] {processor.py:186} INFO - Started process (PID=26546) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T13:02:08.963+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T13:02:08.965+0000] {logging_mixin.py:190} INFO - [2025-09-07T13:02:08.964+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T13:02:09.284+0000] {logging_mixin.py:190} INFO - [2025-09-07T13:02:09.283+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T13:02:09.285+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T13:02:09.294+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.335 seconds
[2025-09-07T13:54:07.971+0000] {processor.py:186} INFO - Started process (PID=26589) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T13:54:07.972+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T13:54:07.974+0000] {logging_mixin.py:190} INFO - [2025-09-07T13:54:07.974+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T13:54:08.359+0000] {logging_mixin.py:190} INFO - [2025-09-07T13:54:08.358+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T13:54:08.360+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T13:54:08.369+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.403 seconds
[2025-09-07T13:54:38.432+0000] {processor.py:186} INFO - Started process (PID=26634) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T13:54:38.433+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T13:54:38.434+0000] {logging_mixin.py:190} INFO - [2025-09-07T13:54:38.434+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T13:54:38.777+0000] {logging_mixin.py:190} INFO - [2025-09-07T13:54:38.775+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T13:54:38.778+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T13:54:38.789+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.360 seconds
[2025-09-07T13:55:09.793+0000] {processor.py:186} INFO - Started process (PID=26676) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T13:55:09.794+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T13:55:09.795+0000] {logging_mixin.py:190} INFO - [2025-09-07T13:55:09.795+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T13:55:10.238+0000] {logging_mixin.py:190} INFO - [2025-09-07T13:55:10.236+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T13:55:10.239+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T13:55:10.246+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.456 seconds
[2025-09-07T13:55:40.934+0000] {processor.py:186} INFO - Started process (PID=26718) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T13:55:40.935+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T13:55:40.937+0000] {logging_mixin.py:190} INFO - [2025-09-07T13:55:40.937+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T13:55:41.330+0000] {logging_mixin.py:190} INFO - [2025-09-07T13:55:41.329+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T13:55:41.330+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T13:55:41.337+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.408 seconds
[2025-09-07T13:56:12.047+0000] {processor.py:186} INFO - Started process (PID=26763) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T13:56:12.047+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T13:56:12.048+0000] {logging_mixin.py:190} INFO - [2025-09-07T13:56:12.048+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T13:56:12.444+0000] {logging_mixin.py:190} INFO - [2025-09-07T13:56:12.443+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T13:56:12.445+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T13:56:12.451+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.408 seconds
[2025-09-07T13:56:43.232+0000] {processor.py:186} INFO - Started process (PID=26804) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T13:56:43.233+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T13:56:43.234+0000] {logging_mixin.py:190} INFO - [2025-09-07T13:56:43.234+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T13:56:43.633+0000] {logging_mixin.py:190} INFO - [2025-09-07T13:56:43.632+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T13:56:43.633+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T13:56:43.640+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.412 seconds
[2025-09-07T13:57:14.343+0000] {processor.py:186} INFO - Started process (PID=26849) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T13:57:14.343+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T13:57:14.344+0000] {logging_mixin.py:190} INFO - [2025-09-07T13:57:14.344+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T13:57:14.731+0000] {logging_mixin.py:190} INFO - [2025-09-07T13:57:14.730+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T13:57:14.732+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T13:57:14.738+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.399 seconds
[2025-09-07T13:57:45.565+0000] {processor.py:186} INFO - Started process (PID=26890) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T13:57:45.565+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T13:57:45.567+0000] {logging_mixin.py:190} INFO - [2025-09-07T13:57:45.567+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T13:57:45.970+0000] {logging_mixin.py:190} INFO - [2025-09-07T13:57:45.968+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T13:57:45.970+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T13:57:45.977+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.416 seconds
[2025-09-07T13:58:16.721+0000] {processor.py:186} INFO - Started process (PID=26935) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T13:58:16.722+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T13:58:16.723+0000] {logging_mixin.py:190} INFO - [2025-09-07T13:58:16.723+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T13:58:17.112+0000] {logging_mixin.py:190} INFO - [2025-09-07T13:58:17.111+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T13:58:17.112+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T13:58:17.120+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.401 seconds
[2025-09-07T13:58:47.797+0000] {processor.py:186} INFO - Started process (PID=26976) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T13:58:47.798+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T13:58:47.799+0000] {logging_mixin.py:190} INFO - [2025-09-07T13:58:47.799+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T13:58:48.225+0000] {logging_mixin.py:190} INFO - [2025-09-07T13:58:48.223+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T13:58:48.225+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T13:58:48.233+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.439 seconds
[2025-09-07T13:59:18.436+0000] {processor.py:186} INFO - Started process (PID=27022) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T13:59:18.436+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T13:59:18.438+0000] {logging_mixin.py:190} INFO - [2025-09-07T13:59:18.437+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T13:59:18.824+0000] {logging_mixin.py:190} INFO - [2025-09-07T13:59:18.823+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T13:59:18.824+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T13:59:18.831+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.399 seconds
[2025-09-07T13:59:49.511+0000] {processor.py:186} INFO - Started process (PID=27065) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T13:59:49.511+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T13:59:49.512+0000] {logging_mixin.py:190} INFO - [2025-09-07T13:59:49.512+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T13:59:49.888+0000] {logging_mixin.py:190} INFO - [2025-09-07T13:59:49.887+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T13:59:49.888+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T13:59:49.895+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.388 seconds
[2025-09-07T14:00:20.597+0000] {processor.py:186} INFO - Started process (PID=27108) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:00:20.597+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T14:00:20.599+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:00:20.598+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:00:20.981+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:00:20.980+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T14:00:20.982+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:00:20.989+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.396 seconds
[2025-09-07T14:00:52.000+0000] {processor.py:186} INFO - Started process (PID=27149) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:00:52.001+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T14:00:52.002+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:00:52.002+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:00:52.317+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:00:52.316+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T14:00:52.318+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:00:52.326+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.329 seconds
[2025-09-07T14:01:23.267+0000] {processor.py:186} INFO - Started process (PID=27194) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:01:23.268+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T14:01:23.270+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:01:23.269+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:01:23.590+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:01:23.588+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T14:01:23.590+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:01:23.599+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.336 seconds
[2025-09-07T14:01:54.268+0000] {processor.py:186} INFO - Started process (PID=27237) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:01:54.269+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T14:01:54.271+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:01:54.270+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:01:54.592+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:01:54.591+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T14:01:54.593+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:01:54.600+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.337 seconds
[2025-09-07T14:02:25.192+0000] {processor.py:186} INFO - Started process (PID=27280) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:02:25.192+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T14:02:25.194+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:02:25.194+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:02:25.571+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:02:25.570+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T14:02:25.571+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:02:25.580+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.392 seconds
[2025-09-07T14:02:56.222+0000] {processor.py:186} INFO - Started process (PID=27323) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:02:56.223+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T14:02:56.224+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:02:56.224+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:02:56.544+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:02:56.542+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T14:02:56.544+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:02:56.553+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.337 seconds
[2025-09-07T14:03:27.230+0000] {processor.py:186} INFO - Started process (PID=27366) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:03:27.230+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T14:03:27.232+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:03:27.232+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:03:27.552+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:03:27.550+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T14:03:27.553+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:03:27.561+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.335 seconds
[2025-09-07T14:03:58.187+0000] {processor.py:186} INFO - Started process (PID=27409) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:03:58.187+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T14:03:58.189+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:03:58.189+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:03:58.529+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:03:58.528+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T14:03:58.529+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:03:58.537+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.355 seconds
[2025-09-07T14:04:29.209+0000] {processor.py:186} INFO - Started process (PID=27452) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:04:29.209+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T14:04:29.211+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:04:29.211+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:04:29.551+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:04:29.549+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T14:04:29.551+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:04:29.559+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.355 seconds
[2025-09-07T14:05:00.281+0000] {processor.py:186} INFO - Started process (PID=27495) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:05:00.281+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T14:05:00.285+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:05:00.284+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:05:00.634+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:05:00.632+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T14:05:00.634+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:05:00.645+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.371 seconds
[2025-09-07T14:05:31.441+0000] {processor.py:186} INFO - Started process (PID=27538) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:05:31.442+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T14:05:31.443+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:05:31.443+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:05:31.791+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:05:31.790+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T14:05:31.792+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:05:31.802+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.365 seconds
[2025-09-07T14:06:02.567+0000] {processor.py:186} INFO - Started process (PID=27581) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:06:02.567+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T14:06:02.569+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:06:02.569+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:06:02.969+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:06:02.968+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T14:06:02.970+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:06:02.977+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.416 seconds
[2025-09-07T14:06:33.114+0000] {processor.py:186} INFO - Started process (PID=27624) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:06:33.115+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T14:06:33.116+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:06:33.116+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:06:33.515+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:06:33.513+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T14:06:33.515+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:06:33.523+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.413 seconds
[2025-09-07T14:07:04.190+0000] {processor.py:186} INFO - Started process (PID=27667) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:07:04.191+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T14:07:04.193+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:07:04.192+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:07:04.589+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:07:04.588+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T14:07:04.589+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:07:04.596+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.410 seconds
[2025-09-07T14:07:35.240+0000] {processor.py:186} INFO - Started process (PID=27710) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:07:35.241+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T14:07:35.242+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:07:35.242+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:07:35.698+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:07:35.697+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T14:07:35.698+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:07:35.705+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.469 seconds
[2025-09-07T14:08:05.769+0000] {processor.py:186} INFO - Started process (PID=27753) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:08:05.769+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T14:08:05.771+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:08:05.771+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:08:06.170+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:08:06.169+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T14:08:06.170+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:08:06.177+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.412 seconds
[2025-09-07T14:08:36.941+0000] {processor.py:186} INFO - Started process (PID=27796) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:08:36.941+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T14:08:36.943+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:08:36.943+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:08:37.344+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:08:37.342+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T14:08:37.344+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:08:37.351+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.414 seconds
[2025-09-07T14:09:08.002+0000] {processor.py:186} INFO - Started process (PID=27839) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:09:08.003+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T14:09:08.004+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:09:08.004+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:09:08.410+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:09:08.408+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T14:09:08.410+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:09:08.417+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.418 seconds
[2025-09-07T14:09:39.087+0000] {processor.py:186} INFO - Started process (PID=27882) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:09:39.088+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T14:09:39.090+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:09:39.089+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:09:39.485+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:09:39.484+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T14:09:39.485+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:09:39.492+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.409 seconds
[2025-09-07T14:10:10.170+0000] {processor.py:186} INFO - Started process (PID=27925) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:10:10.170+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T14:10:10.172+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:10:10.172+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:10:10.569+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:10:10.567+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T14:10:10.569+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:10:10.577+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.411 seconds
[2025-09-07T14:10:41.260+0000] {processor.py:186} INFO - Started process (PID=27968) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:10:41.260+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T14:10:41.262+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:10:41.262+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:10:41.655+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:10:41.654+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T14:10:41.656+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:10:41.663+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.408 seconds
[2025-09-07T14:11:12.430+0000] {processor.py:186} INFO - Started process (PID=28011) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:11:12.430+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T14:11:12.431+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:11:12.431+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:11:12.742+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:11:12.741+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T14:11:12.742+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:11:12.750+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.323 seconds
[2025-09-07T14:11:43.425+0000] {processor.py:186} INFO - Started process (PID=28054) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:11:43.426+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T14:11:43.427+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:11:43.427+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:11:43.754+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:11:43.753+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T14:11:43.755+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:11:43.763+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.343 seconds
[2025-09-07T14:12:14.394+0000] {processor.py:186} INFO - Started process (PID=28097) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:12:14.394+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T14:12:14.396+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:12:14.396+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:12:14.717+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:12:14.716+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T14:12:14.718+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:12:14.726+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.336 seconds
[2025-09-07T14:12:45.392+0000] {processor.py:186} INFO - Started process (PID=28140) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:12:45.392+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T14:12:45.394+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:12:45.394+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:12:45.772+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:12:45.770+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T14:12:45.772+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:12:45.780+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.393 seconds
[2025-09-07T14:13:16.351+0000] {processor.py:186} INFO - Started process (PID=28183) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:13:16.352+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T14:13:16.353+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:13:16.353+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:13:16.678+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:13:16.677+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T14:13:16.679+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:13:16.687+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.341 seconds
[2025-09-07T14:13:47.329+0000] {processor.py:186} INFO - Started process (PID=28226) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:13:47.329+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T14:13:47.331+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:13:47.331+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:13:47.656+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:13:47.655+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T14:13:47.656+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:13:47.665+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.341 seconds
[2025-09-07T14:14:18.287+0000] {processor.py:186} INFO - Started process (PID=28269) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:14:18.287+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T14:14:18.289+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:14:18.289+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:14:18.628+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:14:18.627+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T14:14:18.629+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:14:18.637+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.355 seconds
[2025-09-07T14:14:49.395+0000] {processor.py:186} INFO - Started process (PID=28312) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:14:49.395+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T14:14:49.397+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:14:49.397+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:14:49.720+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:14:49.719+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T14:14:49.720+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:14:49.729+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.338 seconds
[2025-09-07T14:15:20.510+0000] {processor.py:186} INFO - Started process (PID=28355) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:15:20.511+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T14:15:20.512+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:15:20.512+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:15:20.840+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:15:20.839+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T14:15:20.841+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:15:20.849+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.343 seconds
[2025-09-07T14:15:51.613+0000] {processor.py:186} INFO - Started process (PID=28398) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:15:51.614+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T14:15:51.616+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:15:51.615+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:15:52.016+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:15:52.015+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T14:15:52.016+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:15:52.024+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.415 seconds
[2025-09-07T14:16:22.127+0000] {processor.py:186} INFO - Started process (PID=28441) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:16:22.128+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T14:16:22.129+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:16:22.129+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:16:22.553+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:16:22.551+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T14:16:22.553+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:16:22.560+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.437 seconds
[2025-09-07T14:16:53.225+0000] {processor.py:186} INFO - Started process (PID=28484) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:16:53.226+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T14:16:53.227+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:16:53.227+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:16:53.687+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:16:53.686+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T14:16:53.688+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:16:53.696+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.476 seconds
[2025-09-07T14:17:23.778+0000] {processor.py:186} INFO - Started process (PID=28527) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:17:23.779+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T14:17:23.780+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:17:23.780+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:17:24.173+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:17:24.172+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T14:17:24.174+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:17:24.181+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.407 seconds
[2025-09-07T14:17:54.938+0000] {processor.py:186} INFO - Started process (PID=28570) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:17:54.939+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T14:17:54.942+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:17:54.942+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:17:55.402+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:17:55.400+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T14:17:55.403+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:17:55.412+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.480 seconds
[2025-09-07T14:18:25.526+0000] {processor.py:186} INFO - Started process (PID=28613) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:18:25.526+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T14:18:25.528+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:18:25.528+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:18:25.988+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:18:25.987+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T14:18:25.989+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:18:25.998+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.476 seconds
[2025-09-07T14:18:56.162+0000] {processor.py:186} INFO - Started process (PID=28656) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:18:56.164+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T14:18:56.166+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:18:56.165+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:18:56.581+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:18:56.580+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T14:18:56.582+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:18:56.589+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.433 seconds
[2025-09-07T14:19:26.748+0000] {processor.py:186} INFO - Started process (PID=28699) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:19:26.748+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T14:19:26.750+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:19:26.750+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:19:27.180+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:19:27.179+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T14:19:27.181+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:19:27.191+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.448 seconds
[2025-09-07T14:19:57.438+0000] {processor.py:186} INFO - Started process (PID=28742) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:19:57.438+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T14:19:57.440+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:19:57.440+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:19:57.875+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:19:57.873+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T14:19:57.875+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:19:57.882+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.450 seconds
[2025-09-07T14:20:28.342+0000] {processor.py:186} INFO - Started process (PID=28785) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:20:28.343+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T14:20:28.345+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:20:28.345+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:20:28.775+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:20:28.774+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T14:20:28.775+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:20:28.783+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.448 seconds
[2025-09-07T14:20:59.471+0000] {processor.py:186} INFO - Started process (PID=28828) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:20:59.472+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T14:20:59.473+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:20:59.473+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:20:59.849+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:20:59.848+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T14:20:59.850+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:20:59.858+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.389 seconds
[2025-09-07T14:21:30.383+0000] {processor.py:186} INFO - Started process (PID=28871) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:21:30.383+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T14:21:30.385+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:21:30.385+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:21:30.714+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:21:30.713+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T14:21:30.715+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:21:30.723+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.347 seconds
[2025-09-07T14:22:00.985+0000] {processor.py:186} INFO - Started process (PID=28914) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:22:00.986+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T14:22:00.988+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:22:00.988+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:22:01.314+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:22:01.312+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T14:22:01.314+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:22:01.323+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.344 seconds
[2025-09-07T14:22:31.638+0000] {processor.py:186} INFO - Started process (PID=28957) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:22:31.639+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T14:22:31.641+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:22:31.641+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:22:31.975+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:22:31.973+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T14:22:31.975+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:22:31.983+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.352 seconds
[2025-09-07T14:23:02.260+0000] {processor.py:186} INFO - Started process (PID=29000) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:23:02.261+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T14:23:02.263+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:23:02.263+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:23:02.587+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:23:02.586+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T14:23:02.588+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:23:02.596+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.342 seconds
[2025-09-07T14:23:32.872+0000] {processor.py:186} INFO - Started process (PID=29043) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:23:32.873+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T14:23:32.875+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:23:32.875+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:23:33.201+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:23:33.200+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T14:23:33.201+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:23:33.210+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.344 seconds
[2025-09-07T14:24:03.475+0000] {processor.py:186} INFO - Started process (PID=29086) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:24:03.476+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T14:24:03.478+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:24:03.478+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:24:03.803+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:24:03.802+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T14:24:03.804+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:24:03.812+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.343 seconds
[2025-09-07T14:24:34.161+0000] {processor.py:186} INFO - Started process (PID=29129) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:24:34.162+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T14:24:34.164+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:24:34.164+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:24:34.492+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:24:34.490+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T14:24:34.492+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:24:34.501+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.345 seconds
[2025-09-07T14:25:04.776+0000] {processor.py:186} INFO - Started process (PID=29173) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:25:04.776+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T14:25:04.778+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:25:04.778+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:25:05.099+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:25:05.098+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T14:25:05.100+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:25:05.109+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.338 seconds
[2025-09-07T14:25:35.363+0000] {processor.py:186} INFO - Started process (PID=29216) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:25:35.364+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T14:25:35.366+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:25:35.366+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:25:35.700+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:25:35.699+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T14:25:35.701+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:25:35.710+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.354 seconds
[2025-09-07T14:26:06.473+0000] {processor.py:186} INFO - Started process (PID=29259) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:26:06.474+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T14:26:06.476+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:26:06.476+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:26:06.881+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:26:06.880+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T14:26:06.882+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:26:06.889+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.420 seconds
[2025-09-07T14:26:37.071+0000] {processor.py:186} INFO - Started process (PID=29302) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:26:37.072+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T14:26:37.074+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:26:37.073+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:26:37.496+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:26:37.495+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T14:26:37.497+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:26:37.504+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.438 seconds
[2025-09-07T14:27:08.233+0000] {processor.py:186} INFO - Started process (PID=29345) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:27:08.234+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T14:27:08.235+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:27:08.235+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:27:08.640+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:27:08.639+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T14:27:08.640+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:27:08.647+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.418 seconds
[2025-09-07T14:27:38.884+0000] {processor.py:186} INFO - Started process (PID=29388) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:27:38.885+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T14:27:38.887+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:27:38.887+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:27:39.316+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:27:39.315+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T14:27:39.316+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:27:39.325+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.447 seconds
[2025-09-07T14:28:09.521+0000] {processor.py:186} INFO - Started process (PID=29431) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:28:09.522+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T14:28:09.524+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:28:09.524+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:28:09.929+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:28:09.928+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T14:28:09.929+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:28:09.937+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.424 seconds
[2025-09-07T14:28:40.185+0000] {processor.py:186} INFO - Started process (PID=29474) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:28:40.186+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T14:28:40.189+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:28:40.189+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:28:40.589+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:28:40.588+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T14:28:40.589+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:28:40.596+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.417 seconds
[2025-09-07T14:29:10.805+0000] {processor.py:186} INFO - Started process (PID=29517) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:29:10.806+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T14:29:10.809+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:29:10.808+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:29:11.208+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:29:11.207+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T14:29:11.208+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:29:11.215+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.416 seconds
[2025-09-07T14:29:41.863+0000] {processor.py:186} INFO - Started process (PID=29560) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:29:41.864+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T14:29:41.866+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:29:41.865+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:29:42.258+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:29:42.257+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T14:29:42.259+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:29:42.266+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.407 seconds
[2025-09-07T14:42:58.277+0000] {processor.py:186} INFO - Started process (PID=29626) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:42:58.279+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T14:42:58.281+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:42:58.281+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:42:58.691+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:42:58.689+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T14:42:58.691+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:42:58.699+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.425 seconds
[2025-09-07T14:43:29.801+0000] {processor.py:186} INFO - Started process (PID=29671) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:43:29.802+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T14:43:29.803+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:43:29.803+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:43:30.254+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:43:30.252+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T14:43:30.254+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:43:30.268+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.471 seconds
[2025-09-07T14:44:01.158+0000] {processor.py:186} INFO - Started process (PID=29712) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:44:01.158+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T14:44:01.160+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:44:01.159+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:44:01.593+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:44:01.592+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T14:44:01.594+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:44:01.602+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.447 seconds
[2025-09-07T14:44:32.042+0000] {processor.py:186} INFO - Started process (PID=29755) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:44:32.044+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T14:44:32.046+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:44:32.046+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:44:32.369+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:44:32.368+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T14:44:32.369+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:44:32.377+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.339 seconds
[2025-09-07T14:45:03.233+0000] {processor.py:186} INFO - Started process (PID=29800) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:45:03.234+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T14:45:03.236+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:45:03.235+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:45:03.556+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:45:03.555+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T14:45:03.557+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:45:03.564+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.335 seconds
[2025-09-07T14:45:34.386+0000] {processor.py:186} INFO - Started process (PID=29843) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:45:34.387+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T14:45:34.389+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:45:34.389+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:45:34.732+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:45:34.731+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T14:45:34.732+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:45:34.741+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.360 seconds
[2025-09-07T14:46:05.479+0000] {processor.py:186} INFO - Started process (PID=29886) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:46:05.480+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T14:46:05.481+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:46:05.481+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:46:05.840+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:46:05.839+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T14:46:05.841+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:46:05.849+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.375 seconds
[2025-09-07T14:46:36.484+0000] {processor.py:186} INFO - Started process (PID=29929) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:46:36.485+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T14:46:36.486+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:46:36.486+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:46:36.810+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:46:36.809+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T14:46:36.811+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:46:36.819+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.339 seconds
[2025-09-07T14:47:07.444+0000] {processor.py:186} INFO - Started process (PID=29972) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:47:07.444+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T14:47:07.446+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:47:07.446+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:47:07.788+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:47:07.787+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T14:47:07.789+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:47:07.801+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.363 seconds
[2025-09-07T14:47:38.469+0000] {processor.py:186} INFO - Started process (PID=30015) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:47:38.470+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T14:47:38.472+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:47:38.471+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:47:38.805+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:47:38.804+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T14:47:38.805+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:47:38.813+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.348 seconds
[2025-09-07T14:48:09.534+0000] {processor.py:186} INFO - Started process (PID=30058) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:48:09.535+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T14:48:09.536+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:48:09.536+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:48:09.864+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:48:09.863+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T14:48:09.864+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:48:09.873+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.343 seconds
[2025-09-07T14:48:40.601+0000] {processor.py:186} INFO - Started process (PID=30101) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:48:40.602+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T14:48:40.604+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:48:40.603+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:48:40.957+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:48:40.956+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T14:48:40.958+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:48:40.967+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.371 seconds
[2025-09-07T14:49:11.698+0000] {processor.py:186} INFO - Started process (PID=30144) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:49:11.699+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T14:49:11.701+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:49:11.700+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:49:12.074+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:49:12.073+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T14:49:12.075+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:49:12.083+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.389 seconds
[2025-09-07T14:49:42.802+0000] {processor.py:186} INFO - Started process (PID=30187) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:49:42.803+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T14:49:42.806+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:49:42.806+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:49:43.237+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:49:43.235+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T14:49:43.237+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:49:43.244+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.447 seconds
[2025-09-07T14:50:13.914+0000] {processor.py:186} INFO - Started process (PID=30230) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:50:13.914+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T14:50:13.916+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:50:13.916+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:50:14.328+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:50:14.327+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T14:50:14.329+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:50:14.336+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.426 seconds
[2025-09-07T14:50:44.474+0000] {processor.py:186} INFO - Started process (PID=30273) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:50:44.475+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T14:50:44.477+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:50:44.477+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:50:44.896+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:50:44.895+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T14:50:44.897+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:50:44.904+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.433 seconds
[2025-09-07T14:51:15.569+0000] {processor.py:186} INFO - Started process (PID=30316) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:51:15.570+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T14:51:15.571+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:51:15.571+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:51:15.972+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:51:15.971+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T14:51:15.973+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:51:15.980+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.415 seconds
[2025-09-07T14:51:46.648+0000] {processor.py:186} INFO - Started process (PID=30359) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:51:46.649+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T14:51:46.650+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:51:46.650+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:51:47.049+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:51:47.048+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T14:51:47.049+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:51:47.056+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.413 seconds
[2025-09-07T14:52:17.759+0000] {processor.py:186} INFO - Started process (PID=30402) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:52:17.759+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T14:52:17.761+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:52:17.761+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:52:18.152+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:52:18.151+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T14:52:18.153+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:52:18.159+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.405 seconds
[2025-09-07T14:52:48.870+0000] {processor.py:186} INFO - Started process (PID=30445) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:52:48.871+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T14:52:48.872+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:52:48.872+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:52:49.264+0000] {logging_mixin.py:190} INFO - [2025-09-07T14:52:49.263+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T14:52:49.265+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T14:52:49.271+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.405 seconds
[2025-09-07T15:46:21.536+0000] {processor.py:186} INFO - Started process (PID=30486) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T15:46:21.536+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T15:46:21.538+0000] {logging_mixin.py:190} INFO - [2025-09-07T15:46:21.538+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T15:46:21.930+0000] {logging_mixin.py:190} INFO - [2025-09-07T15:46:21.929+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T15:46:21.930+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T15:46:21.938+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.405 seconds
[2025-09-07T16:53:29.928+0000] {processor.py:186} INFO - Started process (PID=30529) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T16:53:29.929+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T16:53:29.931+0000] {logging_mixin.py:190} INFO - [2025-09-07T16:53:29.931+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T16:53:30.319+0000] {logging_mixin.py:190} INFO - [2025-09-07T16:53:30.317+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T16:53:30.319+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T16:53:30.326+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.401 seconds
[2025-09-07T18:00:11.661+0000] {processor.py:186} INFO - Started process (PID=30572) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T18:00:11.661+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T18:00:11.663+0000] {logging_mixin.py:190} INFO - [2025-09-07T18:00:11.663+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T18:00:12.120+0000] {logging_mixin.py:190} INFO - [2025-09-07T18:00:12.118+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T18:00:12.121+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T18:00:12.129+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.473 seconds
[2025-09-07T19:39:28.060+0000] {processor.py:186} INFO - Started process (PID=30617) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T19:39:28.061+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T19:39:28.063+0000] {logging_mixin.py:190} INFO - [2025-09-07T19:39:28.063+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T19:39:28.420+0000] {logging_mixin.py:190} INFO - [2025-09-07T19:39:28.419+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T19:39:28.421+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T19:39:28.429+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.374 seconds
[2025-09-07T21:22:47.834+0000] {processor.py:186} INFO - Started process (PID=30658) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T21:22:47.835+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T21:22:47.836+0000] {logging_mixin.py:190} INFO - [2025-09-07T21:22:47.836+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T21:22:48.178+0000] {logging_mixin.py:190} INFO - [2025-09-07T21:22:48.176+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T21:22:48.178+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T21:22:48.187+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.357 seconds
[2025-09-07T22:47:18.172+0000] {processor.py:186} INFO - Started process (PID=30701) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T22:47:18.173+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T22:47:18.174+0000] {logging_mixin.py:190} INFO - [2025-09-07T22:47:18.174+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T22:47:18.496+0000] {logging_mixin.py:190} INFO - [2025-09-07T22:47:18.494+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T22:47:18.496+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T22:47:18.505+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.336 seconds
[2025-09-07T23:58:28.724+0000] {processor.py:186} INFO - Started process (PID=30744) to work on /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T23:58:28.725+0000] {processor.py:914} INFO - Processing file /opt/airflow/dags/simulation_preprocessing_dag.py for tasks to queue
[2025-09-07T23:58:28.727+0000] {logging_mixin.py:190} INFO - [2025-09-07T23:58:28.727+0000] {dagbag.py:588} INFO - Filling up the DagBag from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T23:58:29.080+0000] {logging_mixin.py:190} INFO - [2025-09-07T23:58:29.078+0000] {dagbag.py:387} ERROR - Failed to import: /opt/airflow/dags/simulation_preprocessing_dag.py
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/dagbag.py", line 383, in parse
    loader.exec_module(new_module)
  File "<frozen importlib._bootstrap_external>", line 940, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/opt/airflow/dags/simulation_preprocessing_dag.py", line 25, in <module>
    from ida4sims.db import Molecule
ModuleNotFoundError: No module named 'ida4sims.db'
[2025-09-07T23:58:29.080+0000] {processor.py:927} WARNING - No viable dags retrieved from /opt/airflow/dags/simulation_preprocessing_dag.py
[2025-09-07T23:58:29.088+0000] {processor.py:208} INFO - Processing /opt/airflow/dags/simulation_preprocessing_dag.py took 0.368 seconds
